x-pack/plugins/apm/public/components/alerting/ui_components/apm_rule_params_container/index.tsx

```diff
@@ -41,8 +68,63 @@ export function ApmRuleParamsContainer(props: Props) {
           </EuiFlexItem>
         ))}
       </EuiFlexGrid>
+
       {chartPreview}
       <EuiSpacer size="m" />
     </>
   );
 }
+
+function MinimumWindowSizeWarning({
+  minimumWindowSize,
+}: {
+  minimumWindowSize: MinimumWindowSize;
+}) {
+  return (
+    <EuiCallOut
+      title={`Please increase "For the last" to at least ${
```

Comment:
What about using `i18n` and down below?!

Reply:
Good point